<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowboatMainWindow</class>
 <widget class="QMainWindow" name="ShowboatMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Showboat</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../disk backups/RAPTORX/Tweaks/Windows 2000/Icons/SHELL32/Unknown filetype.ico</normaloff>../../disk backups/RAPTORX/Tweaks/Windows 2000/Icons/SHELL32/Unknown filetype.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(183, 230, 255);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="toolTip">
       <string>Delete the current selected entry</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 239, 239);</string>
      </property>
      <property name="text">
       <string>Delete entry</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../disk backups/RAPTORX/Tweaks/Windows 2000/Icons/SHELL32/Forbidden.ico</normaloff>../../disk backups/RAPTORX/Tweaks/Windows 2000/Icons/SHELL32/Forbidden.ico</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="toolTip">
       <string>Commit changes to the database</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 239, 239);</string>
      </property>
      <property name="text">
       <string>Commit changes</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../disk backups/RAPTORX/Tweaks/Windows 2000/Icons/SHELL32/Floppy (2).ico</normaloff>../../disk backups/RAPTORX/Tweaks/Windows 2000/Icons/SHELL32/Floppy (2).ico</iconset>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 239, 239);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Show Title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="showNameUI">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 239, 239);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Enter a show's title...</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 239, 239);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Episodes Watched</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="episodesWatchedUI">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 239, 239);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 239, 239);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Episodes Total</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="episodesTotalUI">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 239, 239);</string>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="testLabelUI">
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 239, 239);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Rating (out of 5.0)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ratingUI">
        <property name="toolTip">
         <string>What do you think of this show?</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 239, 239);</string>
        </property>
        <property name="maximum">
         <double>5.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" rowspan="4">
     <widget class="QListView" name="showListUI">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 239, 239);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Showboat v0.9b - Double-click an entry to modify it.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
